src/PaperG/FirehoundBlob/Scenario.php

```diff
@@ -16,4 +16,8 @@ class Scenario
     const AN_DESKTOP = 'AppNexus-Desktop';
     const AN_MOBILE  = 'AppNexus-Mobile';
     const AN_DESKTOP_MOBILE = 'AppNexus-Desktop-Mobile';
-} 
+
+    public function allValidScenarios() {
+        return [self::FB_MANAGED, self::FB_UNMANAGED, self::AN_DESKTOP, self::AN_MOBILE, self::AN_DESKTOP_MOBILE];
```

Comment:
Another style is to define the list in the class and return the defined list rather than have it defined in a function (but you might want to use the same list elsewhere)